perfkitbenchmarker/publisher.py

```diff
@@ -302,7 +302,7 @@ def _FindConstantMetadataKeys(self, samples):
 
     for sample in samples:
       for k, v in sample['metadata'].iteritems():
-        if len(unique_values.setdefault(k, set())) < 2:
+        if len(unique_values.setdefault(k, set())) < 2 and v.__hash__:
```

Comment:
List values doesn't have hash function. Which causing a bug, when you put a list as a value in metadata. 
Adding __hash__ skip lists and other types doesn't have a hash